test/SourceKit/Diagnostics/remarks.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: env %sourcekitd-test -req=diags %s -- %s -Xfrontend -Rmodule-api-import | %FileCheck %s
+public typealias Foo = String
+
+// CHECK:      {
+// CHECK-NEXT:   key.diagnostics: [
+// CHECK-NEXT:   {
+// CHECK-NEXT:       key.line: 2,
+// CHECK-NEXT:       key.column: 24,
+// CHECK-NEXT:       key.filepath: "SOURCE_DIR/test/SourceKit/Diagnostics/remarks.swift",
```

Comment:
Need both / and \\ for Windows

Reply:
I think that `%/s` might allow us to get away in this case to avoid the regexes.